### ISSUE TITLE
Detect spicetify updates without api

### ISSUE THREAD

**Issue by Protonosgit** [Protonosgit/Spicetify_Installer#24]
**Describe the feature**
Manager is currently using its own api with github token.
Sometimes the api goes down,token expires(only valid for 90days or so) or url is blocked by whitelist.
In future use local update checking which is already done by spicetify itself.
Could come with problems if spotify is not run for a while updates will only be shown on runntime!

**Additional context**
Maybe check from config files or similar
